### PR TITLE
Wishlist and cart page

### PR DESCRIPTION
**Pull request by SatwikaKacham**
1. Made wishlist and carts page.
2. Made small changes to cards according to pages.
3. Made responsiveness to both pages.
4. [CSB-wishlist](https://39xulr.csb.app/)
5. [CSB-cartspage](https://ioekmk.csb.app/)

### CODE REVIEW

File: cart-page/cart-page.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+.cart-bill {
+    display: flex;
+    flex-direction: column;
+    gap: 0.8rem;
+    width: 20rem;
+    border: 1px solid var(--secoundary);
+    box-shadow: 1px 1px 3px var(--secoundary);
+    padding: 0.5rem;
+    border-radius: 0.3rem;
+    /* margin: 2rem auto; */
```
Comment (gadgetfather) [inline cart-page/cart-page.css:123]: You can remove this comment if not needed

File: cart-page/cart-page.html [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+
+                        <span class="material-icons-outlined cross-badge">
+                            close
+                        </span>
+                    </header>
+                </div>
+            </div>
+
+
+
```
Comment (gadgetfather) [inline cart-page/cart-page.html:207]: you remove this spacing as well

File: wishlist-page/wishlist-page.html [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+        </div>
+
+
+
+
+
+
+
+
+
```
Comment (gadgetfather) [inline wishlist-page/wishlist-page.html]: Here as well extra spacing, you can check out beautify in vs code extensions

**Comment by gadgetfather** [comment]
You use ```cursor: pointer``` for dismissing button (in both wishlist and cart), right its showing cursor only. also, you can add a hover state for your add to cart buttons.
wishlist gif

**Review by gadgetfather — APPROVED** [review]
Approving your PR with small suggestions!
Great work @SatwikaKacham  👍

**Review by dipteekhd — APPROVED** [review]

**Pull request merged**